database/PostgreSQL/models/Restaurants.js

```diff
@@ -0,0 +1,80 @@
+const Sequelize = require('sequelize');
+const sequelize = require('../index.js')
+
+const { Model } = Sequelize;
+
+class Restaurants extends Model {}
+
+Restaurants.init({
+  restaurantID: {
```

Comment:
would consider renaming keys in schema using lowercase to allow easier maintenance over time

Reply:
if left like this, think sequelize will auto rename to "restaurant_i_d" which won't be ideal